### PR TITLE
Added institution tags to reservation and loans

### PR DESCRIPTION
**Pull request by cableman**
#### Link to ticket

https://jira.itkdev.dk/browse/ER-1363

#### Description

Add support for institution tags (from adgangsplatformen)

#### Screenshot of the result

N/A

#### Checklist

- [x] My code passes our static analysis suite.
- [x] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist you have to add a comment explaining why this change
should be exempt from the list.

#### Additional comments or questions

N/A

### CODE REVIEW

File: sites/all/modules/publizon/includes/publizon.auth.inc [hunk 1]
```diff
@@ -32,6 +32,11 @@ function publizon_auth_single_sign_on($name, array $extra) {
   $client = PublizonUserClient::getClient();
   $support_id = $client->getSupportId($name, $retailer_id);
   if ($support_id) {
+    // If only one tag is set at adgangsplatformen this is a string, so convert
+    // to array if that's the case.
+    $tags = $extra['attributes']['userPrivilege'] ?? array();
+    $tags = is_array($tags) ? $tags : array($tags);
```
Comment (rimi-itk) [inline sites/all/modules/publizon/includes/publizon.auth.inc:38]: If you know that `$extra['attributes']['userPrivilege']` is an array (llst) or a scalar you can just use `(array)$extra['attributes']['userPrivilege']` to create an array with the scalar as a single element (cf. https://www.php.net/manual/en/language.types.array.php#language.types.array.casting).

**Review by rimi-itk — APPROVED** [review]
Approved with a comment.

File: sites/all/modules/publizon/includes/publizon.auth.inc (hunk 1 above)
Comment (cableman) [reply sites/all/modules/publizon/includes/publizon.auth.inc:38]: Nice trick, will update the code to morrow

Comment (cableman) [reply sites/all/modules/publizon/includes/publizon.auth.inc]: Fixed

**Review by rimi-itk — APPROVED** [review]

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**